runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsPlugin.java

```diff
@@ -388,12 +395,64 @@ public void onGameTick(GameTick event)
 		lastPlayerLocation = client.getLocalPlayer().getWorldLocation();
 	}
 
+	@VisibleForTesting
+	String buildRecoloredNpcMenuEntryTarget(MenuEntry menuEntry, NPC menuNpc)
+	{
+		List<Color> menuEntryColors = extractMenuEntryColors(menuEntry);
+
+		Color menuNpcNameColor = null;
+		if (config.highlightMenuNames() &&
+			highlightedNpcs.stream().anyMatch(npc -> menuNpc.getIndex() == npc.getIndex()))
+		{
+			menuNpcNameColor = config.getHighlightColor();
+		}
+		else
+		{
+			menuNpcNameColor = menuEntryColors.get(0);
+		}
+
+		if (config.dampenDeadNPCs() && menuNpc.isDead())
+		{
+			menuNpcNameColor = menuNpcNameColor.darker();
+		}
+		final String menuNpcNameColorHex = ColorUtil.colorToHexCode(menuNpcNameColor);
+
+		Color menuNpcLevelColor = null;
+		// NPCs without combat levels need to be treated differently.
+		if (menuNpc.getCombatLevel() > 0)
+		{
+			menuNpcLevelColor = menuEntryColors.get(1);
+			String menuNpcLevelColorHex = ColorUtil.colorToHexCode(menuNpcLevelColor);
+			return String.format(NPC_MENU_ENTRY_TARGET_FORMAT,
+				menuNpcNameColorHex, menuNpc.getName(),
+				menuNpcLevelColorHex, menuNpc.getCombatLevel());
```

Comment:
Generally, multiline arguments passed to a method should be one per line, unless there is a logical coupling (say, argument pairs) that would dictate otherwise. Format strings do not have any such logical coupling, so this should be as follows:
```suggestion
				menuNpcNameColorHex,
				menuNpc.getName(),
				menuNpcLevelColorHex,
				menuNpc.getCombatLevel());
```